### PR TITLE
Use prefixes feature

### PR DESCRIPTION
**Pull request by Simaris**
I moved the content of rdf_property_list.rb into rdf_prperty.rb to reduce redundancy.  That way, I needed to define the prefix resolution just once. Also, both files had similar content, so separation isn't necessarily wanted.

### CODE REVIEW

**Comment by white-gecko** [comment]
Please rebase on current develop.
Also it would be good to have a small example using the prefix feature.

**Comment by white-gecko** [comment]
Make `input.page.data["rdf_prefix_map"]` a parameter of `rdf_resolve_prefix`.
URIs should be identified with `<…>`.

**Comment by Simaris** [comment]
I think the rebasing is done now, so a fast-forward merge should be possible

File: lib/jekyll/filters/rdf_sparql_query.rb [hunk 1]
```diff
@@ -57,7 +60,14 @@ def sparql_query(input, query, test = false)
           hsh.collect{|k,v| [k.to_s, v]}.to_h
         end
         return result
+      rescue SPARQL::Client::ClientError => ce
+        Jekyll.logger.error("malformed query found: \n #{query} \n Error Message: #{ce.message}")
```
Comment (white-gecko) [inline lib/jekyll/filters/rdf_sparql_query.rb]: Maybe this should be reworded

**Comment by white-gecko** [comment]
You should also implement the `rdf_resolve_prefix` function to distinguish between URIs and prefixed-qnames looking for <…>.

**Comment by white-gecko** [comment]
Is superseded by #34

**Pull request closed without merging**